### DISCUSSION TITLE
Code-View Copy Permalink Drops Line Number

### DISCUSSION THREAD

**Discussion by AlecRosenbaum** [community/community#40272] [Repositories]
### Select Topic Area

Bug

### Body

The "Copy Permalink" button no longer (consistently) copies the selected line number.

Just with a random public repository, for example:
![image](https://user-images.githubusercontent.com/1953412/204298071-70c859c6-68d9-4ade-a6d8-a69c0a61eeea.png)

Clicking "Copy Permalink" occasionally drops the selected line number, resulting in `https://github.com/closeio/tasktiger/blob/38f5f174aaca7706b732a46a3b67f5034a66b5cc/tasktiger/worker.py`

**Comment by ossanna16** [comment]
Hi there @AlecRosenbaum and welcome to our community! Thank you for reporting this bug!

**Comment by AlecRosenbaum** [comment]
Aha, I can now consistently reproduce this bug. If you open a link already with the lines selected, then `Copy Permalink` does not copy the link with the selected lines.

For example using the repo above: [link](https://github.com/closeio/tasktiger/blob/38f5f174aaca7706b732a46a3b67f5034a66b5cc/tasktiger/worker.py#L286)

If that link is opened directly, it'll correctly highlight the line (and the url correctly reflects the selected line number). But if `Copy Permalink` is selected on the highlighted line then it copies just the link to the file instead of the link with the line number.

My use case that makes this super noticable/annoying is that my IDE has an "Open in Github" integration where it'll open a git-tracked file within github. I frequently use this when communicating with colleagues, creating github issues, etc. With this bug, I have to copy from the URL directly instead of using buttons within the code view.
